warehouse/query-core/src/main/java/datawave/query/iterator/logic/TermFrequencyExcerptIterator.java

```diff
@@ -66,6 +65,19 @@ public class TermFrequencyExcerptIterator implements SortedKeyValueIterator<Key,
     // the top value
     protected Value tv;
 
+    private float origHalfSize;
+    // the list of hit terms
+    protected ArrayList<String> hitTermsList;
+    // the direction for the excerpt
+    private String direction;
+
+    private boolean trim;
+
+    private static final String BEFORE = "BEFORE";
+    private static final String AFTER = "AFTER";
+    private static final String BOTH = "BOTH";
```

Comment:
This could be an enum

Reply:
```suggestion
    private enum Direction {
        BEFORE,
        AFTER,
        BOTH
    };
```
There will be other things to fix if you take this suggestion.

Reply:
I noticed that there's another class `ExcerptFields` that uses this notion of direction as well - and that uses a String too. It might make sense for these two classes to use the same enum, but that would be a larger refactoring.